services/web/server/src/simcore_service_webserver/resource_manager/garbage_collector.py

```diff
@@ -563,7 +568,7 @@ async def remove_all_projects_for_user(app: web.Application, user_id: int) -> No
                 "The project can be removed as is not shared with write access with other users"
             )
             try:
-                await delete_project_from_db(app, project_uuid, user_id)
+                await delete_project(app, project_uuid, user_id)
```

Comment:
@GitHK @sanderegg  the new naming makes clear that the entire project is now deleted